src/OpenLoco/Input/MouseInput.cpp

```diff
@@ -1853,10 +1853,8 @@ namespace OpenLoco::Input
             oldWindow->callPrepareDraw();
 
             Ui::widget_t* oldWidget = &oldWindow->widgets[widgetIdx];
-            if (
-                oldWidget->type == Ui::widget_type::wt_16 || oldWidget->type == Ui::widget_type::wt_10 || oldWidget->type == Ui::widget_type::wt_9)
+            if (oldWidget->type == Ui::widget_type::wt_10 || oldWidget->type == Ui::widget_type::wt_9)
```

Comment:
why this change?

Reply:
Widget type 16 was unused, but is now the groupbox. I initially tried making it 30, shifting widget end, but apparently we're not ready to change that.

Reply:
okay ignore my wt_16 comments then. Should add a comment that we have repurposed wt_16 somewhere so that if for some magical reason we need it we know what happened.